content/docs/atom-guide/look-dev/_index.md

```diff
@@ -14,4 +14,4 @@ This section contains the information about creative tools and concepts for look
 | [Textures](textures/) | Learn about how textures are supported and processed in Atom. |
 | [Shading](shaders/) | Learn about shading in Atom. |
 | [Color Management](color-management/) | Learn about the color space conversion workflow used in Atom. |
-| [Material Types and Shaders](get-started-materialtypes-and-shaders/) | A beginner-level tutorial introducing material types and shaders in Atom. Learn how to create a custom material type with a simple AZSL shader. | 
\ No newline at end of file
+| [Tutorials](../learning-guide/tutorials/rendering/) | Tutorials to get started on creating custom additions and changes. |
```

Comment:
Did you mean to link the new content/docs/atom-guide/look-dev/tutorials.md here instead? It's a new doc in the same folder so I would have expected it to be linked in this _index.md

Reply:
Or was adding content/docs/atom-guide/look-dev/tutorials.md a mistake? It looks like a duplicate of content/docs/learning-guide/tutorials/rendering/_index.md

Reply:
I made content/docs/atom-guide/look-dev/tutorials.md almost a duplicate of content/docs/learning-guide/tutorials/rendering/_index.md so that there could be a link to the tutorials in the navigation on the left under Atom Look Dev.

I do agree it is a little weird though, I can change content/docs/atom-guide/look-dev/tutorials.md to just have a sentence saying to visit content/docs/learning-guide/tutorials/rendering/_index.md instead of having the same content?

Reply:
Yeah, I think it should just link to the real tutorials page, not be a duplicate. It's less maintenance that way.

Reply:
Sounds good, I made content/docs/atom-guide/look-dev/_index.md link to content/docs/atom-guide/look-dev/tutorials.md, which then has one sentence linking to content/docs/learning-guide/tutorials/rendering/_index.md.